src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$number-of-days: 31;
+$gap-between-cells: 1px;
+$cell-with-gap: 101px;
```

Comment:
Better store gap value separately. And add proper cell width. Sum of these values = cell with gap.